guide/07-working-with-bigdata/02-making-your-data-accessible-to-ga-server.ipynb

```diff
@@ -0,0 +1,838 @@
+{
```

Comment:
<p>GeoAnalytics Server allows you to register datasets in a format called a <a href="http://enterprise.arcgis.com/en/server/latest/get-started/windows/what-is-a-big-data-file-share.htm" target="_blank">big data file share</a>. Big data file shares are items on your Web GIS that can reference data in any of the following data sources:    </p><ul><li>File Share - a directory of datasets stored locally or shared across a network</li><li>HDFS - a <a href="https://hadoop.apache.org/docs/r1.2.1/hdfs_design.html#Introduction" target="_blank">Hadoop Distributed File System</a> directory of datasets &nbsp;</li><li><a href="https://hive.apache.org/" target="_blank">Apache Hive</a> - a metastore database &nbsp; </li><li>Cloud Store - an <a href="https://azure.microsoft.com/en-us/services/storage/blobs/" target="_blank">Azure Blob Storage</a> container or <a href="https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingBucket.html" target="_blank">Amazon Web Services S3 bucket</a>  </li></ul><p> </p><p>Storing your data in a big data file share datastore benefits you because:</p><ul><li>The GeoAnalytics tools read your data only when they are executed, which allows you to update or add data to these locations. &nbsp; </li><li>You can use partitioned data as a single dataset. &nbsp; </li><li>Big data file shares are flexible in how time and geometry are defined, allowing data in multiple formats in a single dataset. </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1076/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Minor change in the first paragraph, and can we change the lists to bullet points?</p>